src/Agent.Sdk/Util/ExceptionsUtil.cs

```diff
@@ -0,0 +1,23 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+using Microsoft.VisualStudio.Services.Agent;
+
+namespace Agent.Sdk.Util
+{
+    public class ExceptionsUtil
+    {
+        public static void HandleAggregateException(AggregateException e, Action<string> action)
```

Comment:
Nit: please specify more exact name for action delegate - probably something like traceErrorAction?

Reply:
Done. Renamed it.